Libplanet/Net/Swarm.cs

```diff
@@ -790,12 +790,11 @@ await foreach (var pair in demandBlockHashes)
                                 {
                                     cancellationToken.ThrowIfCancellationRequested();
 
-                                    // FIXME: Shouldn't we turn off renderBlocks option in IBD?
                                     workspace.Append(
                                         deltaBlock,
                                         DateTimeOffset.UtcNow,
                                         evaluateActions: false,
-                                        renderBlocks: true,
+                                        renderBlocks: false,
```

Comment:
This change should be written in the changelog.

Reply:
Added to the changelog. 681faad